### PR TITLE
add missing  doc subsection

### PR DESCRIPTION
**Pull request by stenwt**
## Describe the Change

Adds missing subsection for "build" subcommand for spectro cli

## Review Changes

💻 [Add Preview URL]()

🎫 [Jira Ticket]()

### CODE REVIEW

File: docs/docs-content/registries-and-packs/spectro-cli-reference.md [hunk 1]
```diff
@@ -107,6 +107,46 @@
...
+
+```bash
+ spectro pack build [PACK_NAME] [TARGET_DIR]
+```
+
+</TabItem>
+
+<TabItem label="Usage" value="cli_build_example">
+```bash
+ spectro pack build my-awesome-pack ./my-awesome-pack
```
[the remark this answers is not in the document]
Comment (stenwt) [reply docs/docs-content/registries-and-packs/spectro-cli-reference.md:130]: spectro as in spectro cli

File: docs/docs-content/registries-and-packs/spectro-cli-reference.md [hunk 2]
```diff
@@ -107,6 +107,46 @@
...
+```
+
+</TabItem>
+
+</Tabs>
+
+### Args
```
[the remark this answers is not in the document]
Comment (stenwt) [reply docs/docs-content/registries-and-packs/spectro-cli-reference.md:137]: yes

**Comment by stenwt** [comment]
Found a missing breaking space; anything else before I push that?

**Review by karl-cardenas-coding — APPROVED** [review]
@stenwt I did minor cosmetic touches. It still needs much attention but this page will get deprecated once we make the official statement.

File: docs/docs-content/registries-and-packs/spectro-cli-reference.md [hunk 3]
```diff
@@ -90,42 +89,77 @@ Authenticate user with Spectro Cloud pack registry by using the login command:
...
 -i, --insecure - Insecure is used when the pack registry is installed in HTTP or HTTPS with self-signed certificates.
 
 -d, --default - Set the server as default Spectro Cloud pack registry for all the CLI commands.
 
-**Note:** In case of HTTPS, if you have access to the pack registry's CA certificate, there is no need for the insecure flag; simply place the CA certificate at /etc/spectro/certs.d/[SERVER]/ca.crt.
 
-</TabItem>
+:::tip
+
+If you have access to the pack registry's Certificate Authority (CA) certificate .Place the CA certificate in the **/etc/spectro/certs.d/[SERVER]/ca.crt** folder. Replace the [SERVER] with the pack registry server name.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/registries-and-packs/spectro-cli-reference.md:109]:
```suggestion
If you have access to the pack registry's Certificate Authority (CA) certificate. Place the CA certificate in the **/etc/spectro/certs.d/[SERVER]/ca.crt** folder. Replace the [SERVER] with the pack registry server name.
```

**Comment by karl-cardenas-coding** [comment]
Failed backports addressed through cherry-picks

**Pull request merged**